lib/src/models/subreddit.dart

```diff
@@ -423,14 +420,80 @@ class Subreddit extends SubredditRef with RedditBaseInitializedMixin {
   }
 }
 
-// TODO(bkonyi): implement.
-// Provides functions to interact with the special [Subreddit]'s filters.
-/*class SubredditFilters {
-  final Subreddit _subreddit;
-  SubredditFilters(this._subreddit) {
-    throw new DRAWUnimplementedError();
+/// Provides functions to interact with the special [Subreddit]'s filters.
+class SubredditFilters {
+  static final RegExp _userRegExp = new RegExp(r'{user}');
+  static final RegExp _specialRegExp = new RegExp(r'{special}');
+  final SubredditRef _subreddit;
+
+  SubredditFilters._(this._subreddit) {
+    if ((_subreddit.displayName != 'all') &&
+        (_subreddit.displayName != 'mod')) {
+      throw new DRAWArgumentError(
+          'Only special Subreddits can be filtered (r/all or r/mod)');
+    }
+  }
+
+  /// Returns a stream of all filtered subreddits.
+  Stream<SubredditRef> call() async* {
+    final user = await _subreddit.reddit.user.me();
+    final path = apiPath['subreddit_filter_list']
+        .replaceAll(_specialRegExp, _subreddit.displayName)
+        .replaceAll(_userRegExp, user.displayName);
+    final Multireddit response = await _subreddit.reddit.get(path);
```

Comment:
Does `Multireddit` work here? I haven't finished it yet.

Reply:
Yes, since `Multireddit`'s `data` array is populated and it's wired into the `Objector`. It's very basic without your functionality ;)